### ISSUE TITLE
Configuration path on Linux is set to an hidden folder within a hidden folder (`.config/.rt64` instead of `.config/rt64`)

### ISSUE THREAD

**Issue by Calinou** [rt64/rt64#56]
[rt64_user_paths.cpp](https://github.com/rt64/rt64/blob/main/src/common/rt64_user_paths.cpp) seems to have an issue with how it's defining the configuration path on Linux. The path is set to `~/.config/.rt64` (notice the dot before `rt64`), which makes discovery more difficult.

PS: As per the [XDG Base Directory specification](https://wiki.archlinux.org/title/XDG_Base_Directory), it's preferred to read the `XDG_CONFIG_HOME` environment variable first if it exists (and use `$XDG_CONFIG_HOME/rt64/`), then fall back to `~/.config/rt64/` and then `~/.rt64/`.

**Comment by DarioSamo** [comment]
Makes sense to me, I probably started off the original plan with the dot and just added the .config without thinking I needed to change that one as well.

**Comment by Rosalie241** [comment]
> Also, I'm curious if RT64 within RMG should use its own path to be consistent with the GlideN64 configuration file path (which is within ~/.config/RMG). This is intuitively where I'd expect the configuration file to be.

Yes, this has been on my mind aswell a little, but rt64 isn't coming to RMG anytime soon in a release, it's pretty far off right now to be usable enough for end-users in my opinion.
